python/tvm/topi/nn/conv2d_transpose.py

```diff
@@ -98,9 +98,9 @@ def declaration_conv2d_transpose_impl(data, kernel, strides, padding, out_dtype,
 
     out_h = simplify(in_h - filter_h + 1 + output_padding[0])
     out_w = simplify(in_w - filter_w + 1 + output_padding[1])
-    dc = tvm.reduce_axis((0, in_c), name='dc')
-    dh = tvm.reduce_axis((0, filter_h), name='dh')
-    dw = tvm.reduce_axis((0, filter_w), name='dw')
+    dc = tvm.te.reduce_axis((0, in_c), name='dc')
+    dh = tvm.te.reduce_axis((0, filter_h), name='dh')
+    dw = tvm.te.reduce_axis((0, filter_w), name='dw')
```

Comment:
seems we can omit "tvm" and do `te.reduce_axis`. can we add test case to prevent from regression?